modules/gmysqlbackend/smysql.cc

```diff
@@ -38,6 +38,22 @@
 typedef bool my_bool;
 #endif
 
+class MySQLThreadCloser
```

Comment:
this black magic code should have a comment on the "why" :)

Reply:
Yes, agreed!